src/Microsoft.Data.Sqlite/Interop/NativeMethods.cs

```diff
@@ -243,15 +255,30 @@ public static int sqlite3_prepare_v2(Sqlite3Handle db, string zSql, out Sqlite3S
         }
 
         [DllImport("sqlite3", CallingConvention = CallingConvention.Cdecl)]
+        public static extern int sqlite3_reset(Sqlite3StmtHandle stmt);
+
+        [DllImport("sqlite3", CallingConvention = CallingConvention.Cdecl)]
         public static extern int sqlite3_step(Sqlite3StmtHandle stmt);
 
+        public static int sqlite3_step_blocking(Sqlite3Handle db, Sqlite3StmtHandle stmt, int milliseconds)
```

Comment:
It does feel a little weird to have implementation here that doesn't deal with native interop. We may want to move this closer to the places that use it. Or just in a different utility class. But it's also fine to just leave it here for now.